### PR TITLE
Setup drone

### PR DESCRIPTION
**Pull request by spinillos**
It needs to setup drone in the project.

Since we don't have enough things and we don't generate tags and/or release versions it only checks the lint and do the tests.

### CODE REVIEW

**Review by undef1nd — APPROVED** [review]
Seems fine to me, but let's wait for Joan to confirm.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.PHONY: lint, test, drone
```
Comment (joanlopez) [inline Makefile:1]: It's the first time I see `.PHONY` targets separated with commas (instead of just white spaces), is that accepted as well? 🤔

**Review by joanlopez — APPROVED** [review]
Looks good, I just left one question! Thanks! :)

File: Makefile (hunk 1 above)
Comment (spinillos) [reply Makefile:1]: Ups... but yes... it was working 😅

**Pull request merged**